### PR TITLE
*: Support grant stmt in new plan.

### PR DESCRIPTION
**Pull request by shenli**
@coocood @qiuyesuifeng

### CODE REVIEW

File: executor/converter/convert_stmt.go [hunk 1]
```diff
@@ -918,3 +919,4 @@ func convertGrant(converter *expressionConverter, v *ast.GrantStmt) (*stmts.Gran
 	}
 	return oldGrant, nil
 }
+*/
```
Comment (coocood) [inline executor/converter/convert_stmt.go]: Remove those lines.

File: executor/converter/convert_stmt.go [hunk 2]
```diff
@@ -875,6 +875,7 @@ func convertPrivElem(converter *expressionConverter, v *ast.PrivElem) (*coldef.P
 	return oldPrim, nil
 }
 
+/*
```
Comment (qiuyesuifeng) [inline executor/converter/convert_stmt.go]: Can this function be removed?

File: executor/grant.go [hunk 3]
```diff
@@ -0,0 +1,529 @@
...
+		return errors.Trace(err)
+	}
+	ok, err := dbUserExists(e.ctx, user, host, db.Name.O)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	if ok {
+		return nil
+	}
+	// Entry does not exists for user-host-db. Insert a new entry.
```
Comment (qiuyesuifeng) [inline executor/grant.go]: does not exist

**Comment by shenli** [comment]
PTAL @coocood @qiuyesuifeng

**Comment by coocood** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**